### DISCUSSION TITLE
AZW3R file issues

### DISCUSSION THREAD

**Discussion by K2AAE** [noDRM/DeDRM_tools#270] [General]
I was having issues downloading the last book in a series with my Kindle PC, it would keep telling me I had to upgrade the version of Kind PC. I sent the book to my Kindle Touch but the file was named AZW3R and is not seen by Calibre so I can't open it.

**Comment by ElleKayEm** [comment]
The azw3r is not the actual book.  There should be a file with the extension azw3.  Look in the documents folder, not in the book's sdr folder.

**Reply by K2AAE** [reply]
Thanks, I found the AZW3 file but when I tried to convert it got the DRM error, I think I need to install the Kindle serial number somewhere...

**Reply by ElleKayEm** [reply]
Yup, romad275's comment tells you where.  I'll just add that copy and paste of the serial number from your Content & Devices page at Amazon will save you from entering it wrong.

**Comment by romad275** [comment]
Kevin, I went to  Calibre>Preferences>Advanced>Plugins>File Type Plugins>DeDRM>Customize Plugin>eink Kindle ebooks

Sincerely,
Dennis B. Swaney

Cogito Ergo Mac

**Reply by K2AAE** [reply]
I poked it long enough to find it, got the serial number installed and everything works, the two series last books have been imported, finally. I tried a couple of other series that are a couple of years old and they have no issues, I think it is just the current published books they are jerking around for Kindle Unlimited user. Thanks and I hope the help catches up to the existing version of Calibre.
